plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -962,6 +969,12 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
         tableProperties.put("numFiles", "-1");
         tableProperties.put("totalSize", "-1");
 
+        // Extra properties
+        Map<String, String> extraProperties = HiveTableProperties.getExtraProperties(tableMetadata.getProperties());
```

Comment:
nit: How about switching to static import HiveTableProperties.getExtraProperties

Reply:
Yep, done!